spec/features/users/session_timeout_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+
+RSpec.describe 'Session timeout', type: :feature do
+  it 'expires the session after 15 minutes' do
+    @user = User.create(email: 'user@example.com', password: 'password')
+
+    visit root_path
+
+    click_on 'Login'
+
+    fill_in 'Email', with: @user.email
+    fill_in 'Password', with: @user.password
+
+    click_on 'Login'
+
+    expect(page).to have_current_path(root_path)
+
+    expect(page).to have_content('Log Out')
+    
+    Timecop.travel(15.minutes.from_now)
+
+    visit root_path
+
```

Comment:
Interesting, looking forward to seeing how we might be able to automatically redirect.